archivy/api.py

```diff
@@ -124,13 +124,34 @@ def update_dataobj(dataobj_id):
     """
     if request.json.get("content"):
         try:
-            data.update_item(dataobj_id, request.json.get("content"))
+            data.update_item_md(dataobj_id, request.json.get("content"))
             return Response(status=200)
         except BaseException:
             return Response(status=404)
     return Response("Must provide content parameter", status=401)
 
 
+@api_bp.route("/dataobjs/frontmatter/<int:dataobj_id>", methods=["PUT"])
+def update_dataobj_frontmatter(dataobj_id):
+    """
+    Updates frontmatter of object of given id.
+
+    Paramter in JSON body:
+
+    - **content**: markdown text of new dataobj.
```

Comment:
We need to change this from content -> title. Do you think we should also be able to edit the tags this way?

Reply:
Thing with tags is that a `tags.split(",")` is not robust on its own and the user needs to know that commas are separators (as opposed to spaces for example). Maybe when work on the tag systems starts there might be a nicer way (autosuggest existing tags and stuff like that). The API supports changing the tags already.